<details>
<summary>Pre-Commit Test failed! Click here</summary>

```
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for case conflicts.................................................Passed
Check JSON...........................................(no files to check)Skipped
Check for broken symlinks............................(no files to check)Skipped
Detect Private Key.......................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

manifests/base/routes/argocd_server_route.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/base/services/argocd_server_metrics_service_patch.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/rbac/argocd-manager-role.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)
  7:3       error    wrong indentation: expected 4 but found 2  (indentation)
  9:3       error    wrong indentation: expected 4 but found 2  (indentation)
  11:3      error    wrong indentation: expected 4 but found 2  (indentation)
  13:3      error    wrong indentation: expected 4 but found 2  (indentation)
  15:3      error    wrong indentation: expected 4 but found 2  (indentation)

manifests/overlays/dev/projects/projects.yaml
  1:1       warning  missing document start "---"  (document-start)
  7:3       error    wrong indentation: expected 4 but found 2  (indentation)
  10:3      error    wrong indentation: expected 4 but found 2  (indentation)

manifests/base/services/argocd_metrics_service_patch.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/serviceaccounts/argocd-dex-server.yaml
  1:1       warning  missing document start "---"  (document-start)
  10:81     error    line too long (94 > 80 characters)  (line-length)

manifests/crds/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/secrets/auth/argocd-secret.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  9:81      error    line too long (207 > 80 characters)  (line-length)
  10:81     error    line too long (160 > 80 characters)  (line-length)
  11:81     error    line too long (1432 > 80 characters)  (line-length)
  18:81     error    line too long (288 > 80 characters)  (line-length)
  20:8      error    too many spaces after hyphen  (hyphens)
  20:5      error    wrong indentation: expected 8 but found 4  (indentation)

manifests/overlays/dev/projects/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

examples/argocd-manager-rolebinding.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/crds/aggregate-to-admin.yaml
  1:1       warning  missing document start "---"  (document-start)
  8:1       error    wrong indentation: expected 2 but found 0  (indentation)
  9:3       error    wrong indentation: expected 4 but found 2  (indentation)
  11:3      error    wrong indentation: expected 4 but found 2  (indentation)
  14:3      error    wrong indentation: expected 4 but found 2  (indentation)
  30:1      error    wrong indentation: expected 2 but found 0  (indentation)
  31:3      error    wrong indentation: expected 4 but found 2  (indentation)
  33:3      error    wrong indentation: expected 4 but found 2  (indentation)
  35:3      error    wrong indentation: expected 4 but found 2  (indentation)

manifests/overlays/dev/rbac/argocd-manager-role-binding.yaml
  1:1       warning  missing document start "---"  (document-start)
  10:1      error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  5:1       error    wrong indentation: expected 2 but found 0  (indentation)
  11:1      error    wrong indentation: expected 2 but found 0  (indentation)
  25:1      error    wrong indentation: expected 2 but found 0  (indentation)
  29:1      error    wrong indentation: expected 2 but found 0  (indentation)

manifests/base/serviceaccounts/argocd-manager.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/secrets/clusters/secret-generator.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/applications/dev_app/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/secrets/gpg/secret-generator.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)

.aicoe-ci.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/overlays/dev/applications/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)

.sops.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/base/deployments/argocd-repo-server_patch.yaml
  1:1       warning  missing document start "---"  (document-start)
  9:7       error    wrong indentation: expected 8 but found 6  (indentation)
  12:9      error    wrong indentation: expected 10 but found 8  (indentation)
  15:9      error    wrong indentation: expected 10 but found 8  (indentation)
  22:7      error    wrong indentation: expected 8 but found 6  (indentation)

manifests/overlays/dev/secrets/clusters/.sops.yaml
  1:1       warning  missing document start "---"  (document-start)

manifests/base/deployments/argocd-server_patch.yaml
  1:1       warning  missing document start "---"  (document-start)
  9:7       error    wrong indentation: expected 8 but found 6  (indentation)
  11:9      error    wrong indentation: expected 10 but found 8  (indentation)

manifests/overlays/dev/applications/dev_app/dev-app.yaml
  1:1       warning  missing document start "---"  (document-start)
  16:5      error    wrong indentation: expected 6 but found 4  (indentation)

manifests/overlays/dev/secrets/clusters/dev.cluster.example.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  12:81     error    line too long (227 > 80 characters)  (line-length)
  21:81     error    line too long (288 > 80 characters)  (line-length)
  23:8      error    too many spaces after hyphen  (hyphens)
  23:5      error    wrong indentation: expected 8 but found 4  (indentation)

manifests/overlays/dev/secrets/auth/secret-generator.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/secrets/auth/dex-server-sa.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  15:81     error    line too long (288 > 80 characters)  (line-length)
  17:8      error    too many spaces after hyphen  (hyphens)
  17:5      error    wrong indentation: expected 8 but found 4  (indentation)

manifests/base/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:1       error    wrong indentation: expected 2 but found 0  (indentation)
  8:1       error    wrong indentation: expected 2 but found 0  (indentation)

manifests/overlays/dev/secrets/gpg/secret.enc.yaml
  1:1       warning  missing document start "---"  (document-start)
  6:81      error    line too long (6448 > 80 characters)  (line-length)
  13:81     error    line too long (288 > 80 characters)  (line-length)
  15:8      error    too many spaces after hyphen  (hyphens)
  15:5      error    wrong indentation: expected 8 but found 4  (indentation)
```
</details>

--- a/docs/versions.md
+++ b/docs/versions.md
@@ -1,12 +1,12 @@
 # Versions
 
-ArgoCD: 1.6.1
+ArgoCD: 1.7.4
 
-KSOPs: 2.1.2
+KSOPs: 2.1.4
 
 Kustomize: 3.8.0
 
-SOPS: 3.6.0+
+SOPS: 3.6.0
 
 The KSOPS and Kustomize versions refer to the ones provisioned with ArgoCD.
 

--- a/manifests/base/deployments/argocd-repo-server_patch.yaml
+++ b/manifests/base/deployments/argocd-repo-server_patch.yaml
@@ -7,7 +7,7 @@ spec:
     spec:
       containers:
       - name: argocd-repo-server
-        image: quay.io/aicoe/argocd:v1.6.1-1
+        image: quay.io/aicoe/argocd:v1.7.4-1
         env:
         - name: GNUPGHOME
           value: /home/argocd/.gnupg

--- a/manifests/base/kustomization.yaml
+++ b/manifests/base/kustomization.yaml
@@ -1,7 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- github.com/argoproj/argo-cd/manifests/base?ref=v1.6.1
+- github.com/argoproj/argo-cd/manifests/base?ref=v1.7.4
 - routes/argocd_server_route.yaml
 - serviceaccounts/argocd-manager.yaml
 patchesStrategicMerge:

--- a/manifests/crds/kustomization.yaml
+++ b/manifests/crds/kustomization.yaml
@@ -2,4 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - aggregate-to-admin.yaml
-- github.com/argoproj/argo-cd/manifests/crds?ref=v1.6.1
+- github.com/argoproj/argo-cd/manifests/crds?ref=v1.7.4

